tests/e2e/data/observatory/enrichment/files/polygon-public-none.geojson

```diff
@@ -1,17 +1,12 @@
 {
 "type": "FeatureCollection",
 "features": [
-{ "type": "Feature", "properties": { "do_area": 1883075.52247, "do_geoid": "17031815701", "geoid": "17031815701", "intersected_area": 301279.05714, "one_car": 397.0, "poverty": 101.0, "user_area": 11022717.026860001 }, "geometry": { "type": "Polygon", "coordinates": [ [ [ -87.83843994140625, 41.832990869731084 ], [ -87.750205993652344, 41.807149141688363 ], [ -87.750205993652344, 41.834269892288312 ], [ -87.83843994140625, 41.832990869731084 ] ] ] } },
```

Comment:
Here we have a different number of rows. Did the input change?

Reply:
I have needed to remove some rows of polygon-public-none.geojson since the results are wrong. That means we have a bug in version v1.0.0: when the user adds more than one filter, the results are not filtered